src/shared/services/exporting/favoriteUtils.ts

```diff
@@ -26,6 +26,22 @@ import { Folder } from 'shared/modules/favorites/foldersDuck'
 
 export const CYPHER_FILE_EXTENSION = '.cypher'
 
+export function exportFavoritesAsBigCypherFile(favorites: Favorite[]): void {
```

Comment:
This feature has been requested, and the implementation works. But there's no UI element to call this function yet